src/ensembl/production/metadata/scripts/create_datasets_json.py

```diff
@@ -67,12 +67,14 @@ def main(json_input, release_id, conn_uri, status="Submitted"):
 
 if __name__ == "__main__":
     # Expecting JSON input, release id, and connection URI as command-line arguments
-    if len(sys.argv) < 4:
-        print("Usage: python create_datasets.py <json_input> <release_id> <conn_uri> <status(optional)>")
-        sys.exit(1)
-
-    json_input = sys.argv[1]
-    release_id = sys.argv[2]
-    conn_uri = sys.argv[3]
+    # if len(sys.argv) < 4:
+    #     print("Usage: python create_datasets.py <json_input> <release_id> <conn_uri> <status(optional)>")
+    #     sys.exit(1)
 
+    # json_input = sys.argv[1]
+    # release_id = sys.argv[2]
+    # conn_uri = sys.argv[3]
+    json_input = "/home/danielp/PycharmProjects/ensembl-metadata-api/src/ensembl/production/metadata/scripts/human_pangenome_variation_stats.json"
```

Comment:
This looks like it's intended to be a one-off input. Is it meant to be committed?